kalite/coachreports/hbtemplates/coach_nav/landing.handlebars

```diff
@@ -1,6 +1,6 @@
 <div id="coach_report_wrapper">
 	<div id="summary_mainview">
-		<h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}<button id="show_tabular_report" class="btn btn-success pull-right">{{_ "Show Tabular Report" }}</button></h1>
+		<h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success pull-right">{{_ "Show Tabular Report" }}</button>{{/if}}</h1>
```

Comment:
Conditionally show button based on the existence of recent data, use learner_events as a proxy for this.